data/_ui/tooltips.txt

```diff
@@ -304,19 +307,19 @@ tip "illegal:"
 	`Amount of credits you can be fined for carrying this outfit.`
 
 tip "income:"
-	`Amount of credits you receive per day that you own this outfit, even if it is not in active use.`
+	`Amount of credits received per day when you own this outfit, even if it is not in active use.`
 	
 tip "inertia reduction:"
 	`Reduces effective ship mass. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in inertia, while a total value of 11 only results in a 10 percent reduction.`
 
 tip "ion protection:"
-	`Protection against weapons that do ion damage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in ion damage, while a total value of 11 only results in a 10 percent reduction.`
+	`Protection provided against ion damage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in ion damage, while a total value of 11 only results in a 10 percent reduction.`
 
 tip "scramble protection:"
-	`Protection against weapons that do scrambling damage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in ion damage, while a total value of 11 only results in a 10 percent reduction.`
+	`Protection provided against weapons that do scrambling damage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in scramble damage, while a total value of 11 only results in a 10 percent reduction.`
```

Comment:
This one didn't get the full protection tooltip treatment.

Reply:
```suggestion
	`Protection provided against scrambling damage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in scramble damage, while a total value of 11 only results in a 10 percent reduction.`
```